### PR TITLE
Maintenance of README.rst and package documentation

### PR DESCRIPTION
**Pull request by HolQue**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -12,40 +12,67 @@
...
+   Use the following command to install ``GenPackageDoc``:
+
+   .. code::
+
+      setup.py install
+
+How to use
+----------
+
+The main goal of ``GenPackageDoc`` is to document Python sources that are stored within a repository.
```
Comment (test-fullautomation) [inline README.rst]: Hi Holger,
this is not correct. GenPackageDoc is not documenting.
"GenPackageDoc" provides a toolchain to generate documentation out of ...
Please correct.
Thank you,
 Thomas

**Comment by test-fullautomation** [comment]
Hi Holger,
thank you for your pull request.
Please refer to my finding.
Thank you,
 Thomas

File: README.rst (hunk 1 above)
Comment (HolQue) [reply README.rst]: Fixed.

File: README.rst [hunk 2]
```diff
@@ -60,7 +60,7 @@ How to install
 How to use
 ----------
 
-The main goal of ``GenPackageDoc`` is to document Python sources that are stored within a repository.
+``GenPackageDoc`` provides a toolchain to generate documentation out of Python sources that are stored within a repository.
```
Comment (test-fullautomation) [inline README.rst:63]: Hi Holger,
what about the RST and Latex files wich are used to generate the documentation?
Thank you,
  Thomas

Comment (HolQue) [reply README.rst:63]: They are mentioned explicitely at the top of the README.

**Review by test-fullautomation — APPROVED** [review]
Hi Holger,
looks good to me!
Thank you,
Thomas

**Pull request merged**